addons/survey/models/survey_user_input.py

```diff
@@ -19,6 +19,7 @@ class SurveyUserInput(models.Model):
     _description = "Survey User Input"
     _rec_name = "survey_id"
     _order = "create_date desc"
+    _inherit = ['mail.thread', 'mail.activity.mixin']
```

Comment:
Doesn't that require a migration script ?

Reply:
There are no computed fields to init, so I suppose this should be ok.

However inherits existence depending on saas versions are defined in ``upgrade/migrations/util/_inherit.py``. Not sure if auto generated or updated manually, maybe to check with @nseinlet to be sure ?

Reply:
I checked that file and it seems to be only modified by CHS:
https://github.com/odoo/upgrade/commits/master-website-slides-mark-done-2021-std/migrations/util/_inherit.py

Which made me think it's auto-generated every now and then.

So not sure indeed, if necessary I will gladly modify it, waiting for @nseinlet to enlighten us then :smile: